.github/actions/test_behavior_bin_ofs/action.yaml

```diff
@@ -41,7 +41,8 @@ runs:
           - name: Run Test Ofs
             shell: bash
             working-directory: bin/ofs
-            run: cargo test --features ${{ inputs.feature }} --no-default-features
+            timeout-minutes: 1
```

Comment:
I feel like this timeout is too small

Reply:
I think I need help for this: https://github.com/apache/opendal/actions/runs/9076169655/job/24938329304?pr=4611

Reply:
Weired, no ideas about this. Let's just set at job level instead.

Reply:
It works, thanks